modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -51,7 +52,7 @@ The *Serverless Deployment* option is displayed in the *Import from git* form on
 
 . In the *Pipelines* section, select *Add Pipeline*, and then click *Show Pipeline Visualization* to see the pipeline for the application.
 
-. In the *Advanced Options* section, the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL. You can clear the check box if you do not want to expose your application on a public route.
+. In the *Advanced Options* section, the *Target port* and the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL. You can change the target port if your application uses a different port to expose their data. You can clear the check box if you do not want to expose your application on a public route.
```

Comment:
```suggestion
. Optional: In the *Advanced Options* section, the *Target port* and the *Create a route to the application* is selected by default so that you can access your application using a publicly available URL. This URL uses port [TBD? 80?] by default. 
+
If your application does not expose its data on the default public port, 80, clear the [name of] check box, and set the target port number by [doing TBD]
```
I don't know the operation, but I have the impression it might help to expand it a bit more. This suggestion is a pure guess of what this might look like. What do you think?

Reply:
Otherwise: 
- It looks like you might be using "their" as a gender-neutral possessive for the application, a single inanimate object. Is this anthropomorphism? How about using "its" instead? 
- If the user makes a choice based on a condition, "If...then", does putting the "if" condition first make that choice easier for the user to understand?